src/Uplink/View/Provider.php

```diff
@@ -0,0 +1,21 @@
+<?php declare( strict_types=1 );
+
+namespace StellarWP\Uplink\View;
+
+use League\Plates\Engine;
```

Comment:
I think it is too heavy handed to be introducing a templating engine as a dependency into this uplink library.